### PR TITLE
Add Sponsors Button to GitHub Repo

### PR DESCRIPTION
**Pull request by mroeschke**
While discussion on other support avenues are still ongoing, I think it would still be beneficial to add the a Sponsors Button to the repo that points to https://pandas.pydata.org/donate.html

Note: https://www.flipcause.com/secure/cause_pdetails/MzA3OA== is the direct page to donate but this link doesn't appear as "donate friendly" IMO

Let me know if there's a better link to reference 

Reference to the Support Button on Numpy's page: https://github.com/numpy/numpy/pull/13700

### CODE REVIEW

**Review by gfyoung — APPROVED** [review]

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+custom: https://pandas.pydata.org/donate.html
```
Comment (jreback) [inline .github/FUNDING.yml]: does this need a newline?

Comment (mroeschke) [reply .github/FUNDING.yml]: Looks like other projects have a newline in their yml, will add one.

Comment (jreback) [reply .github/FUNDING.yml]: lol thanks

**Comment by jreback** [comment]
where exactly should the sponsor button appear?

**Comment by mroeschke** [comment]
Looks like we need to enable it in the "Settings" menu of the repository, but I can't immediately find it:

https://help.github.com/en/articles/displaying-a-sponsor-button-in-your-repository

**Comment by mroeschke** [comment]
I may not have enough permissions to access the repository settings. Can you enable it @jreback?

Screen Shot 2019-06-07 at 10 23 01 AM

**Pull request merged**